cabot/cabotapp/models.py

```diff
@@ -727,11 +732,24 @@ def _run(self):
                     result.succeeded = status['succeeded']
             else:
                 result.succeeded = status['succeeded']
-            if not status['succeeded']:
-                if result.error:
-                    result.error += u'; Job "%s" failing on Jenkins' % self.name
+
+            if self.max_build_failures and status['consecutive_failures']:
```

Comment:
This is going to be False if `self.max_build_failures` or `status['consecutive_failures']` is 0, but we want to run the check in that case...could change to `if <x> is not None` or just remove since it should always be True, probably would prefer removing it but no strong opinion

Reply:
would be good to add tests for all these cases as well ([test example](https://github.com/Affirm/cabot/blob/master/cabot/cabotapp/tests/test_status_check.py#L63-L99), [Jenkins test fixture example](https://github.com/Affirm/cabot/blob/master/cabot/cabotapp/tests/fixtures/jenkins_response.json))